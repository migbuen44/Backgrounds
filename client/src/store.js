import { configureStore } from '@reduxjs/toolkit';
import userLoginReducer from './slices/userLoginSlice';
import loginModalReducer from './slices/loginModalSlice';
import userInfoReducer from './slices/userInfoSlice';

export default configureStore({
  reducer: {
    userLogin: userLoginReducer,
    loginModal: loginModalReducer,
    userInfo: userInfoReducer,
  },
});
