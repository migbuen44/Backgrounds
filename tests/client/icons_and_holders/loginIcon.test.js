import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer'
import store from '../../../client/src/store';
import LoginIcon from '../../../client/src/components/icons_and_holders/loginIcon';

describe('LoginIcon', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Provider store={store}>
        <LoginIcon />
      </Provider>, div);
  })

  it('matches snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <LoginIcon />
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  })
})