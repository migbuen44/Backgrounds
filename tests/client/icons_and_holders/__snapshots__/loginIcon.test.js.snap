// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginIcon matches snapshot 1`] = `
<div
  className="lib-icons_and_holders-module-loginContainer-1qYL iconCircle click"
  onClick={[Function]}
>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-user fa-w-14 lib-icons_and_holders-module-loginButton-1mCw"
    data-icon="user"
    data-prefix="fas"
    focusable="false"
    role="img"
    style={
      Object {
        "color": "black",
      }
    }
    viewBox="0 0 448 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</div>
`;
